include/manager.hpp

```diff
@@ -30,15 +31,110 @@ class Manager
      * @param[in] iFace - interface to implement.
      * @param[in] connection - Dbus Connection.
      */
-    Manager(const std::shared_ptr<boost::asio::io_context>& ioCon,
-            const std::shared_ptr<sdbusplus::asio::dbus_interface>& iFace,
-            const std::shared_ptr<sdbusplus::asio::connection>& asioConnection);
+    Manager(
+        const std::shared_ptr<boost::asio::io_context>& ioCon,
+        const std::shared_ptr<sdbusplus::asio::dbus_interface>& operationIFace,
+        const std::shared_ptr<sdbusplus::asio::dbus_interface>& stateIFace,
+        const std::shared_ptr<sdbusplus::asio::connection>& asioConnection);
 
     /**
      * @brief Destructor.
      */
     ~Manager() = default;
 
+    /**
+     * @brief Write on hardware
+     * API can be used to perform keyword writes on VPD EEPROM.
+     *
+     * @param[in] i_input Input to write keyword.
+     *
+     * @return Data which is written successfully.
+     */
+    types::ReturnValType
+        writeKeywordOnSource(const types::WriteKeywordSignature i_input);
```

Comment:
Jinu: Need to club the API's.
- WriteKeyword (Will take care of both write on cache or hardware or both based on an enum passed in the parameter)
- Same for read
- GetLocationCode - We returrn what we have, no input required from parameter.

Reply:
why is this complete flipover now?